torch/_torch_docs.py

```diff
@@ -970,6 +970,64 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.atanh`.
 """)
 
+add_docstr(torch.asarray,
+           r"""
+asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
+
+Creates a new :class:`Tensor` out of :attr:`obj` storage, by either copying or
+aliasing it.
+
+:attr:`obj` will be tested for, in order:
+
+1. a :class:`Tensor` object (copy and alias allowed)
+2. a DLPack object (copy and alias allowed)
+3. a Python object that implements the buffer protocol (copy and alias allowed)
+4. an arbitrary sequence (only copy allowed)
+
+When :attr:`copy` is not given, this function will try first to alias the given
+object. However, it will fallback into copying the elements if needed. If it is
+``False``, only aliasing is allowed (``ValueError`` is raised otherwise), and
+vice-versa.
+
+.. warning::
+    When passing a Python object that implements the buffer protocol, this function
```

Comment:
Good warning. Should this also have a pointer to frombuffer?

Reply:
I have referenced `frombuffer` in the `seealso`section. Do you think I should do it when talking about how it uses the provided `dtype`, too?